### PR TITLE
samples: nrf_desktop: Peer removal on keyboard

### PR DESCRIPTION
**Pull request by Qbicz**
Use Fn key to remove bonds.

Signed-off-by: Filip Kubicz <filip.kubicz@nordicsemi.no>

### CODE REVIEW

**Comment by Qbicz** [comment]
@MarekPieta, can you review?

File: samples/nrf_desktop/configuration/nrf52_pca20037/app_ZDebug.conf [hunk 1]
```diff
@@ -18,6 +18,10 @@ CONFIG_DESKTOP_LED_COLOR_COUNT=1
 
 CONFIG_DESKTOP_POWER_MANAGER_ENABLE=y
 
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0211
```
Comment (MarekPieta) [inline samples/nrf_desktop/configuration/nrf52_pca20037/app_ZDebug.conf]: Maybe add comment, which key is it?

Comment (pdunaj) [reply samples/nrf_desktop/configuration/nrf52_pca20037/app_ZDebug.conf]: discussed offline. I think this is fine as fn key is not used at all at the moment
when we will start using fn key this part will also be modified

**Comment by Qbicz** [comment]
All tests passed. @pdunaj can you merge?

**Pull request merged**